aspen/internal/cluster/cluster.go

```diff
@@ -199,6 +199,7 @@ func (c *Cluster) goFlushStore(sCtx signal.Context) {
 				flush.Flush(sCtx, change.State)
 			}
 		})
+		// HERE
```

Comment:
It seems like "HERE" doesn't provide enough context what this is marking. Can you make it more verbose?